src/librustdoc/clean/inline.rs

```diff
@@ -241,6 +241,7 @@ fn build_external_function(cx: &mut DocContext<'_>, did: DefId) -> clean::Functi
         decl,
         generics,
         header: hir::FnHeader { unsafety: sig.unsafety(), abi: sig.abi(), constness, asyncness },
```

Comment:
note to self: get rid of this later, it can be calculated on-demand from the DefId.

Reply:
Open an issue to not forget it then. ;)